_saas-integrations/saasoptics/saasoptics-v1.md

```diff
@@ -0,0 +1,120 @@
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: SaaSOptics
+permalink: /integrations/saas/saasoptics
+keywords: saasoptics, integration, schema, etl saasoptics, saasoptics etl, saasoptics schema
+layout: singer
+# input: false
+
+key: "saasoptics-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "saasoptics"
+display_name: "SaaSOptics"
+
+singer: true
+
+tap-name: "SaaSOptics"
+repo-url: https://github.com/singer-io/tap-saasoptics
+
+this-version: "1"
+
+api: |
+  [{{ integration.display_name }} REST API](https://saasoptics.zendesk.com/hc/en-us/articles/115003595913-URLs-and-REST-Info){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: false 
+
+historical: "1 year"
+frequency: "1 hour"
+tier: "Free"
+
+api-type: "platform.saasoptics"
+
+anchor-scheduling: true
+cron-scheduling: true
+
+extraction-logs: true
+loading-reports: true
+
+table-selection: true
+column-selection: true
+
+# attribution-window: "# days"
+# attribution-is-configurable: 
+
+# setup-name: ""
+
+
+# -------------------------- #
+#      Feature Summary       #
+# -------------------------- #
+
+feature-summary: |
+  Stitch's {{ integration.display_name }} integration replicates data using the {{ integration.api | flatify | strip }}. Refer to the [Schema](#schema) section for a list of objects available for replication.
+
+
+# -------------------------- #
+#      Setup Instructions    #
+# -------------------------- #
+
+setup-steps:
+  - title: "Identify your account name and server subdomain"
+    anchor: "account-subdomain"
+    content: |
+      The server subdomain and account name are in the URL you use to access your {{ integration.display_name }}. If you are unsure of what your link is, locate it within your welcome email from {{ integration.display_name }}. Your server subdomain is the first portion of the URL, and the account name is between the first and second forward slashes.
+
+      ![SaaSOptics URL format.]({{ site.baseurl }}/images/integrations/saasoptics-url.png){:style="max-width: 550px;"}
+
+  - title: "Obtain your token"
+    anchor: "obtain-token"
+    content: |
+      1. Login to the {{ integration.display_name }} account you want to replicate data from.
+      2. In the upper right hand side of the page, click on the user account button that shows your username.
+      3. Click on **API**.
+
+         ![SaaSOptics API hyperlink.]({{ site.baseurl }}/images/integrations/saasoptics-api-button.png){:style="max-width: 550px;"}
+         {:start="4"}
+      4. Generate an API token and keep note of it - you will use it for the Stitch integration.
```

Comment:
> you will use it for the Stitch integration.

Consider rephrasing - this is what they're aiming to do right now, so it's a bit redundant. Maybe `- you'll need it to complete the next step.` or something similar.